libs/domain/user/address-remove/address-remove.component.ts

```diff
@@ -41,6 +41,7 @@ export class UserAddressRemoveComponent extends I18nMixin(
       open
       .heading=${this.i18n('checkout.address.remove-address')}
       @oryx.close=${this.onClose}
+      @oryx.modal.closed=${this.onClose}
```

Comment:
what is this?

Reply:
A fix. @tolerants changed events's names for modal since >= 1.4, and oryx.close event is deprecated. Probably he forgot to fix this on some components, me as well. Added new events for all modals for backward compatibility